src/Prelude/Linear/Generically.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+module Prelude.Linear.Generically where
+
+-- | A datatype whose instances are defined generically, using the
+-- 'Generics.Linear.Generic' representation.
+-- Generic instances can be derived via @'Generically' A@ using
+-- @-XDerivingVia@.
+newtype Generically a = Generically a
+
```

Comment:
`Generically` and `Generically1` have been [added to `base-4.17`](https://github.com/ghc/ghc/blob/master/libraries/base/changelog.md). I don't know whether this is relevant for you, but I thought I'd mention it just in case.

Reply:
It's definitely relevant. I'm not sure how we will handle this, but when 9.4 is out, we will strive to make these newtypes re-exports from base.